lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -150,6 +172,8 @@ def refresh_key_map(self):
             'storage_type':     f'storage_enclosure>{self.enclosure_id}>type',
             'controller_type':  f'storage_enclosure>{self.enclosure_id}>controller>type',
             'name':             f'storage_enclosure>{self.enclosure_id}>name',
+            'cvg':              f'server_node>{self.machine_id}>storage>cvg_count',
```

Comment:
Should we change the key:
'cvg_number'

Reply:
No, it's in line with the confstore template.
If you are suggesting the change in template then I believe the cvg_count is the appropriate name as it suggests how many cvg entries are there in cvg dictionary.